src/sass/partials/_footer.scss

```diff
@@ -0,0 +1,38 @@
+.footer {
+  background-color: rgba(247, 247, 247, 0.95); //lite green
+  color: rgb(84, 84, 84); //dark green
```

Comment:
we have already variables, use them instead of pasting own colors. If we missing then add them

Reply:
Corrected